### ISSUE TITLE
questiones about the model performance

### ISSUE THREAD

**Issue by ideasplus** [ICEORY/PMF#2]
Hello, thanks you for sharing the code.

I have run the pmf model in semantic kitti training set using the default settings. However, the evaluation result in validation set is lower than the value 63.9 reported in the paper. 
![image](https://user-images.githubusercontent.com/24294571/138803716-0fb87d5a-8932-4cc5-abe5-b1df237dedac.png)

![image](https://user-images.githubusercontent.com/24294571/138841070-3315b408-f8c9-46f8-b70a-9df78a1a8061.png)

Meanwhile, I also run the salsanext model provied in the codebase in semantic kitti training set using the default settings. The evaluation result in validation set is higer than the value 59.4 reported in the paper and **the pmf model.**
![image](https://user-images.githubusercontent.com/24294571/138842654-46ac17b3-6823-4d62-9c80-3ebdb07c8c61.png)

Could you give some suggestions?

**Comment by ideasplus** [comment]
Yes, I use the default batch size value according to here https://github.com/ICEORY/PMF/blob/1875be9bd019a7e8a121d92831fa3cbd557e2ca1/tasks/pmf/config_server_kitti.yaml#L14 
Maybe this is the key to the problem. I have noticed the value is set to 8 for semantic-kitti and 32 for nuscene. Unfortunately I can't further validate this now because I only have one gpu with 16G memory. 

Do you have any plan to release the pretrained model for kitti and nuscene for validation？

**Comment by kingskymoon** [comment]
Hello，my model trained by myself is about 0.6011 miou. Is there ant other tricks？
by the way, how to use the salsanext code ，could you give some suggestions？


> Hello, thanks you for sharing the code.
> 
> I have run the pmf model in semantic kitti training set using the default settings. However, the evaluation result in validation set is lower than the value 63.9 reported in the paper. ![image](https://user-images.githubusercontent.com/24294571/138803716-0fb87d5a-8932-4cc5-abe5-b1df237dedac.png)
> 
> ![image](https://user-images.githubusercontent.com/24294571/138841070-3315b408-f8c9-46f8-b70a-9df78a1a8061.png)
> 
> Meanwhile, I also run the salsanext model provied in the codebase in semantic kitti training set using the default settings. The evaluation result in validation set is higer than the value 59.4 reported in the paper and **the pmf model.** ![image](https://user-images.githubusercontent.com/24294571/138842654-46ac17b3-6823-4d62-9c80-3ebdb07c8c61.png)
> 
> Could you give some suggestions?
